src/main/java/org/jenkinsci/plugins/workflow/libs/LibraryRecord.java

```diff
@@ -45,6 +49,31 @@
         this.changelog = changelog;
     }
 
+    @Exported
+    public String getName() {
+        return name;
+    }
+
+    @Exported
+    public String getVersion() {
+        return version;
+    }
+
+    @Exported
+    public Set<String> getVariables() {
+        return Collections.unmodifiableSet(variables);
+    }
+
+    @Exported
+    public boolean isTrusted() {
+        return trusted;
+    }
+
+    @Exported
+    public boolean isChangelog() {
+        return changelog;
+    }
```

Comment:
This (and `isTrusted` to some extent) do not seem likely to be particularly interesting to external consumers of the REST API, so I could remove the `@Exported` methods for them if anyone feels strongly about it.